src/app/core/models/repo.model.ts

```diff
@@ -34,6 +35,11 @@ export class Repo {
     return false;
   }
 
+  /** Gets org/repo from full http url */
+  private static getRepoUrl(sessionInformation: string) {
+    return sessionInformation.split('/').slice(-2).join('/');
```

Comment:
This works for most cases. In rare cases, users may enter something like `https://github.com/AY2122S1-CS2103T-W15-3/tp/` which would not work. But I am fine with this method of extracting the org/repo.

Reply:
We can enhance this code to handle URLs ending with `/` as well.